environment.yml

```diff
@@ -23,7 +23,7 @@ dependencies:
   - geotiff
   - gmp
   - graphviz
-  - gsl>=2.6,<=2.7.0
+  - gsl>=2.6
   - hdf5
```

Comment:
why is this okay, considering that gsl 2.7.1 was creating issues on CentOS in #4877 ?